app/assets/stylesheets/components/dropdowns.scss

```diff
@@ -26,6 +26,10 @@
       }
     }
 
+    &.blankslate {
+      padding: 0 $padding-base-horizontal $padding-base-vertical $padding-base-horizontal;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `0 $padding-base-horizontal $padding-base-vertical` instead of `0 $padding-base-horizontal $padding-base-vertical $padding-base-horizontal`